Makefile

```diff
@@ -132,17 +144,53 @@ generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Build the docker image
-docker-build: build-linux
-	@for target in $(TARGETS); do                                                      \
-	  image=$(IMAGE_PREFIX)$${target}$(IMAGE_SUFFIX);                                  \
-	  docker build -t $(REGISTRY)/$${image}:$(VERSION)                                 \
-	    --label $(DOCKER_LABELS)                                                       \
-	    -f $(BUILD_DIR)/$${target}/Dockerfile .;                                       \
+docker-build: 
```

Comment:
Maybe we shouldn't change the docker-build here but add a new command like "klever-publish-build"

Reply:
Same for docker push